Utilities/Dialogue Box.gd

```diff
@@ -108,4 +108,11 @@ func Finished():
 	if forceArrow == true:
 		$PauseArrow.show()
 	isFinished = true
-	pass
\ No newline at end of file
+	pass
+
+func parse_string(text: String):
+	var expandedText = expand(text)
+	return expandedText.split("\n")
+
+func expand(text: String):
+	return text.format({"TrainerName": Global.TrainerName})
```

Comment:
Ace wants to stick to dynamic coding so please remove the static type hints.

Godot code standards again. `expanded_text`

Reply:
Fixed